### PR TITLE
Define the service for commands, queries and events on the config file.

### PR DESCRIPTION
**Pull request by vladyslav-fenchak**
┆Issue is synchronized with this [ClickUp task](https://app.clickup.com/t/17nuw8w) by [Unito](https://www.unito.io)

### CODE REVIEW

File: minos/common/configuration/config.py [hunk 1]
```diff
@@ -64,6 +66,7 @@
     "events.queue.database": "MINOS_EVENTS_QUEUE_DATABASE",
     "events.queue.user": "MINOS_EVENTS_QUEUE_USER",
     "events.queue.password": "MINOS_EVENTS_QUEUE_PASSWORD",
+    "events.service": "MINOS_EVENTS_QUEUE_PASSWORD",
```
Comment (garciparedes) [inline minos/common/configuration/config.py:69]:
```suggestion
    "events.service": "MINOS_EVENTS_SERVICE",
    "queries.service": "MINOS_QUERIES_SERVICE",
```

File: minos/common/configuration/config.py [hunk 2]
```diff
@@ -92,6 +95,7 @@
     "commands.queue.database": "commands_queue_database",
     "commands.queue.user": "commands_queue_user",
     "commands.queue.password": "commands_queue_password",
+    "commands.service": "commands_queue_password",
```
Comment (garciparedes) [inline minos/common/configuration/config.py:98]:
```suggestion
    "commands.service": "commands_service",
```

File: minos/common/configuration/config.py [hunk 3]
```diff
@@ -50,6 +51,7 @@
     "commands.queue.database": "MINOS_COMMANDS_QUEUE_DATABASE",
     "commands.queue.user": "MINOS_COMMANDS_QUEUE_USER",
     "commands.queue.password": "MINOS_COMMANDS_QUEUE_PASSWORD",
+    "commands.service": "MINOS_COMMANDS_QUEUE_PASSWORD",
```
Comment (garciparedes) [inline minos/common/configuration/config.py:54]:
```suggestion
    "commands.service": "MINOS_COMMANDS_SERVICE",
```

File: minos/common/configuration/config.py [hunk 4]
```diff
@@ -101,11 +105,12 @@
...
-    "events.queue.host": "events_queue_host",
-    "events.queue.port": "events_queue_port",
-    "events.queue.database": "events_queue_database",
-    "events.queue.user": "events_queue_user",
-    "events.queue.password": "events_queue_password",
+    "events.queue.host": "MINOS_EVENTS_QUEUE_HOST",
+    "events.queue.port": "MINOS_EVENTS_QUEUE_PORT",
+    "events.queue.database": "MINOS_EVENTS_QUEUE_DATABASE",
+    "events.queue.user": "MINOS_EVENTS_QUEUE_USER",
+    "events.queue.password": "MINOS_EVENTS_QUEUE_PASSWORD",
```
Comment (garciparedes) [inline minos/common/configuration/config.py:112]:
```suggestion
    "events.queue.host": "events_queue_host",
    "events.queue.port": "events_queue_port",
    "events.queue.database": "events_queue_database",
    "events.queue.user": "events_queue_user",
    "events.queue.password": "events_queue_password",
```

File: minos/common/configuration/config.py [hunk 5]
```diff
@@ -101,11 +105,12 @@
...
+    "events.service": "events_queue_password",
```
Comment (garciparedes) [inline minos/common/configuration/config.py:113]:
```suggestion
    "events.service": "events_service",
    "queries.service": "queries_service",
```

**Review by garciparedes — CHANGES_REQUESTED** [review]
Hi @vladyslav-fenchak, could you resolve the comments I left over the changes?

**Review by andrea-mucci — APPROVED** [review]

**Review by garciparedes — APPROVED** [review]
Hi @vladyslav-fenchak, it LGTM!

**Pull request merged**